presto-jdbc/pom.xml

```diff
@@ -142,7 +142,13 @@
         <dependency>
             <groupId>io.airlift</groupId>
             <artifactId>concurrent</artifactId>
-            <scope>test</scope>
+            <exclusions>
```

Comment:
It is not required. I checked that on my local environment, and on yet another machine. It is better to exclude this dependency though, as it is not used.